jina/executors/indexers/cache.py

```diff
@@ -43,14 +55,15 @@ def __init__(self, path, logger):
                 self.cache_val_to_id = dict()
 
         def close(self):
+            """Flushes the in-memory cache to pickle files."""
```

Comment:
should super.close be called as well here?

Reply:
It's called by the `DocCache` closing lifecycle